pywren/wrenhandler.py

```diff
@@ -46,7 +55,7 @@ def download_runtime_if_necessary(s3conn, runtime_s3_bucket, runtime_s3_key):
     """
 
     # get runtime etag
-    runtime_meta = s3conn.meta.client.head_object(Bucket=runtime_s3_bucket, 
+    runtime_meta = s3_client.head_object(Bucket=runtime_s3_bucket,
```

Comment:
Did we reach any longer term consensus on this ? If not we can talk about it this week